Gemfile.lock

```diff
@@ -21,10 +22,10 @@ GIT
       xpath (~> 2.0)
 
 GIT
-  remote: git://github.com/plataformatec/devise.git
-  revision: a321282ccc93d73f9d0f7a1319d3fcab27d25b20
+  remote: https://github.com/plataformatec/devise.git
+  revision: ffe9d6d406e79108cf32a2c6a1d0b3828849c40b
   specs:
-    devise (3.5.3)
+    devise (4.0.0.pre.dev)
```

Comment:
@JuanitoFatas This might be issue of test failures? Can we use released gem instead of this?